### DISCUSSION TITLE
Filament Multi-tenancy RegisterTeam: How to ->visible a custom action only if the current authenticated user has at least 1 team?

### DISCUSSION THREAD

**Discussion by devhoussam1998** [filamentphp/filament#12230] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2.61

### How can we help you?

```
    public function getCancelFormAction(): Action
    {
        $user = Filament::auth()->user();
        $teamId = $user ? $user->team_id : null;

        return Action::make('cancel')
            // ->url('https://google.com')
            ->visible(fn () => $teamId !== null);
    }
```

**Comment by micraux** [comment]
If you have defined a `teams()` relationship in your Model:

```php
->visible(fn (): bool => auth()->user()->teams->count() > 1)
```

**Reply by devhoussam1998** [reply]
I setup the `teams()` relationship in my **User** model everything work well.

but `->visible(fn (): bool => auth()->user()->teams->count() > 1)` does not wok even the current authenticated user has already create a team.

**Reply by devhoussam1998** [reply, marked as answer]
fix it with `->visible(fn (): bool => Filament::auth()->user()->teams->count() >= 1)`
